man/check-os-release.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -eu
+
+test -e /etc/os-release && os_release='/etc/os-release' || os_release='/usr/lib/os-release'
+. "${os_release}"
+
+echo "Running on ${PRETTY_NAME:-Linux}"
+
+if [ "${ID:-linux}" = "debian" ] || [ "${ID_LIKE:-}" = "debian" ]; then
```

Comment:
btw, wouldn't the shell scripting cool kids replace `] || [` with `-o` ?

(not a shell scripting cool kid myself though)

Reply:
No, apparently shellcheck recommends going the other way. (Some potential inconsistency in evaluation, but I don't recall the details.)